tests/test.ts

```diff
@@ -101,6 +107,7 @@ describe('Test clasp clone <scriptId> function', () => {
     if (isPR !== 'false') {
       this.skip();
     }
+    fs.writeFileSync('.clasp.json', CLASP_SETTINGS);
```

Comment:
Can you add a comment?

Reply:
Can you extract this line (`fs.writeFileSync`) to remove duplicate code/strings.